internal/controllers/provisioningrequest_controller.go

```diff
@@ -560,93 +547,3 @@ func (t *provisioningRequestReconcilerTask) getCrClusterTemplateRef(ctx context.
 			"a valid (%s) ClusterTemplate does not exist in any namespace",
 			clusterTemplateRefName))
 }
-
-func (r *ProvisioningRequestReconciler) finalizeProvisioningRequest(
-	ctx context.Context, provisioningRequest *provisioningv1alpha1.ProvisioningRequest) error {
-
-	var labels = map[string]string{
-		provisioningRequestNameLabel: provisioningRequest.Name,
-	}
-	listOpts := []client.ListOption{
-		client.MatchingLabels(labels),
-	}
-
-	// Query the NodePool created by this ProvisioningRequest. Delete it if exists.
-	nodePoolList := &hwv1alpha1.NodePoolList{}
-	if err := r.Client.List(ctx, nodePoolList, listOpts...); err != nil {
-		return fmt.Errorf("failed to list node pools: %w", err)
-	}
-	for _, nodePool := range nodePoolList.Items {
-		copiedNodePool := nodePool
-		if err := r.Client.Delete(ctx, &copiedNodePool); client.IgnoreNotFound(err) != nil {
-			return fmt.Errorf("failed to delete node pool: %w", err)
-		}
-	}
-
-	// If the ClusterInstance has been created by this ProvisioningRequest, delete it.
-	// The SiteConfig operator will also delete the namespace.
-	clusterInstanceList := &siteconfig.ClusterInstanceList{}
-	if err := r.Client.List(ctx, clusterInstanceList, listOpts...); err != nil {
-		return fmt.Errorf("failed to list cluster instances: %w", err)
-	}
-	for _, clusterInstance := range clusterInstanceList.Items {
-		copiedClusterInstance := clusterInstance
-		if err := r.Client.Delete(ctx, &copiedClusterInstance); client.IgnoreNotFound(err) != nil {
-			return fmt.Errorf("failed to delete cluster instance: %w", err)
-		}
-	}
-
-	if len(clusterInstanceList.Items) == 0 {
-		// If the ClusterInstance has not been created. Query the namespace created by
-		// this ProvisioningRequest. Delete it if exists.
-		namespaceList := &corev1.NamespaceList{}
-		if err := r.Client.List(ctx, namespaceList, listOpts...); err != nil {
-			return fmt.Errorf("failed to list namespaces: %w", err)
-		}
-		for _, ns := range namespaceList.Items {
-			copiedNamespace := ns
-			if err := r.Client.Delete(ctx, &copiedNamespace); client.IgnoreNotFound(err) != nil {
-				return fmt.Errorf("failed to delete namespace: %w", err)
-			}
-		}
-	}
-
-	return nil
-}
-
-func (r *ProvisioningRequestReconciler) handleFinalizer(
```

Comment:
A clusterscoped owner can own resources across different namespaces. The owner is now set in the policytemplate configmap because the utility function CreateK8sCR was updated to support ownerReference for cluster scoped CR as well in [PR275](https://github.com/openshift-kni/oran-o2ims/pull/275).

Reply:
Right! That's great then. Thank you.